sharktank/sharktank/models/llm/llm.py

```diff
@@ -162,16 +163,34 @@ def prefill(
         if self.inference_norm:
             h *= math.sqrt(h.shape[-1])
 
+        if self.config.attention_chunk_size is not None:
+            chunked_attention_mask = [
+                ops.replicate(
+                    self.chunked_attention_mask(attention_mask[0]),
+                    count=len(self.cache.pipeline_to_device_map),
+                    devices=self.cache.pipeline_to_device_map[pipeline],
+                )
+                for pipeline in range(self.cache.pipeline_count)
+            ]
```

Comment:
```suggestion
        if self.config.attention_chunk_size is not None:
            chunked_attention_mask = [
                ops.replicate(
                    self.chunked_attention_mask(attention_mask[pipeline]),
                    count=len(self.cache.pipeline_to_device_map[pipeline]),
                    devices=self.cache.pipeline_to_device_map[pipeline],
                )
                for pipeline in range(self.cache.pipeline_count)
            ]
```
This will require further work for sharded/pipeline models, but will be fine for now for unsharded.